Updating the README in the context of the MicroProfile component test plan review

fabiobrz opened the pull request:
This is to update and improve READMEs in the context of the MicroProfile component test plan review.
I've removed the `Observability` layer section altogether since it is no klonger valid for EAP 7.4.x

FYI @marekkopecky 
---
Please make sure your PR meets the following requirements:
- [x] Pull Request contains a description of the changes
- [x] Pull Request does not include fixes for multiple issues/topics
- [x] Code is formatted, imports ordered, code compiles and tests are passing
- N/A Link to the passing job is provided - just updates to READMEs, no jobs
- [x] Code is self-descriptive and/or documented
- N/A Description of the tests scenarios is included (see #46) - just updates to READMEs, no tests

marekkopecky commented on README.md:
```diff
@@ -1,122 +1,129 @@
+* [MicroProfile OpenAPI](https://github.com/eclipse/microprofile-open-api)
+* [MicroProfile OpenTracing](https://github.com/eclipse/microprofile-opentracing)
+
+More tests could be added, specifically covering the MicroProfile 
```
This is obviously true, always more tests could be added. But if we need to track this in upstream, I prefer to use github issue https://github.com/jboss-eap-qe/eap-microprofile-test-suite/issues with the description of the details.

fabiobrz replied:
+1, removing.

marekkopecky commented on README.md:
```diff
@@ -1,122 +1,129 @@
 ## Requirements for the testsuite
- - JDK 8+
- - Docker
+- JDK 8+
```
Is this still valid? AFAIK JDK8 is not supported for WF master.

marekkopecky commented on README.md:
```diff
@@ -1,122 +1,129 @@
+[README](./tooling-server-configuration/README.md)
 
 ## Run the testsuite
+Just execute:
```
Hmm, this looks like both commands are necessary, can you please clarify it in the readme?

fabiobrz replied:
Yeah, sure, it can be improved.

marekkopecky commented on README.md:
```diff
@@ -1,122 +1,129 @@
 
 ## Export testing deployments
+Test deployments can be 
+[exported by Arquillian](https://arquillian.org/guides/getting_started_rinse_and_repeat/#export_the_deployment):  
```
same here

fabiobrz replied:
+1

marekkopecky commented on README.md:
```diff
@@ -1,122 +1,129 @@
-- Trigger build by `Build Now` button - NOTE FIRST RUN WILL FAIL DUE TO ISSUES <https://issues.jenkins-ci.org/browse/JENKINS-40574/> <https://issues.jenkins-ci.org/browse/JENKINS-41929> but all the subsequent runs will work
+- Copy content of [Jenkinsfile](./Jenkinsfile) into Pipeline script section and `Save`
+    + Change `linux` label in `agent { label 'linux' }` to reflect the label assigned to your Jenkins nodes
+    + Change `java-1.8.0-oracle` in `jdk 'java-1.8.0-oracle'` to a lable matching an installed JDK
```
Java11?

fabiobrz replied:
Right, fixing.

marekkopecky commented on README.md:
```diff
@@ -1,122 +1,129 @@
-
-In order to test the _just_ this reduced set of MicroProfile specs, then, an 
-additional property should be provided to the command lines described earlier, 
-namely `-Dmp.specs.scope=observability`.
```
Should we remove observability from pom files as well? Or at least create an issue to do it later as a followup task?

fabiobrz replied:
Exactly, #249 logged.

marekkopecky commented on tooling-server-configuration/README.md:
```diff
@@ -1,22 +1,26 @@
+## Configure the tested server instance
+
+To achieve the desired server configuration you'd most likely want to implement the `MicroProfileServerSetupTask` 
+interface, which extends `ServerSetupTask`, and annotate a test class with the `@ServerSetup` annotation.
```
extends -> implements?

marekkopecky commented:
I have a few minor comments, otherwise LGTM.

FYI @OndrejKotek @mnovak1 If you would like to review this, definitely feel free to do so.

mnovak1 commented on README.md:
```diff
@@ -1,122 +1,129 @@
+- [microprofile-jwt](./microprofile-jwt) - contains MicroProfile Config deployments and tests
+- [microprofile-metrics](./microprofile-metrics) - contains MicroProfile Config deployments and tests
+- [microprofile-open-api](./microprofile-open-api) - contains MicroProfile Config deployments and tests
+- [microprofile-opentracing](./microprofile-opentracing) - contains MicroProfile Config deployments and tests
```
It seems there is copy paste error in above lines - `...MicroProfile Config...`

fabiobrz replied:
+1, I removed the section as per @marekkopecky [comment](https://github.com/jboss-eap-qe/eap-microprofile-test-suite/pull/248#discussion_r859612846)

fabiobrz commented on README.md:
```diff
@@ -1,122 +1,129 @@
+
+## Modules
+
+### MicroProfile specs testing modules
```
+1, fixing.

fabiobrz commented:
Hi @marekkopecky and @mnovak1 - thanks for reviewing. I have addressed your comments, feel free to resolve the threads or provide additional feedback.

marekkopecky commented:
Thank you! All comments have been solved, I'm merging it.
